package/MDAnalysis/coordinates/CRD.py

```diff
@@ -151,12 +153,23 @@ def __init__(self, filename, **kwargs):
         ----------
         filename : str or :class:`~MDAnalysis.lib.util.NamedStream`
              name of the output file or a stream
+
+        crdext   : bool (optional)
+             By default, noextended CRD format is used [``False``].
+             However, extended CRD format can be forced by
+             specifying `crdext` ``=True``.
         """
 
         self.filename = util.filename(filename, ext='crd')
         self.crd = None
 
-    def write(self, selection, frame=None):
+        # account for explicit crd format, if requested
+        if "crdext" in kwargs.keys():
+            self.crdext = kwargs.pop('crdext')
+        else:
+            self.crdext = False   # if not requested, default to NOEXT format
+
+    def write(self, selection, frame=None, crdext=False):
```

Comment:
I would think that `extended=False` is only really needed in init. Why would you want to override the format choice when writing?
```suggestion
    def write(self, selection, frame=None):
```

I am open for a discussion, though, just make your case for having the `extended` argument in write(), too.